doc/source/operations/rocky-linux-9.rst

```diff
@@ -0,0 +1,604 @@
+synced to the local Pulp server. This includes host packages (Deb/RPM) and
+container images.
+
+To sync host packages:
+
+.. code-block:: console
+
+   kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/pulp-repo-sync.yml
+   kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/pulp-repo-publish.yml
+
+Once the host package content has been tested in a test/staging environment, it
+may be promoted to production:
+
+.. code-block:: console
+
+   kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/pulp-repo-promote-production.yml
+
+To sync container images:
+
+.. code-block:: console
+
+   kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/pulp-container-sync.yml
+   kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/pulp-container-publish.yml
+
+Build locally customised container images
+-----------------------------------------
+
+.. note::
+
+   The container images are provided by StackHPC Release Train are
+   suitable for most deployments. In this case, this step can be skipped.
+
+In some cases it is necessary to build some or all images locally to apply
+customisations. In order to do this it is necessary to set
+``stackhpc_pulp_sync_for_local_container_build`` to ``true`` before
+syncing container images.
+
+To build the overcloud images locally and push them to the local Pulp server:
+
+.. code-block:: console
+
+   kayobe overcloud container image build --push
+
+It is possible to build a specific set of images by supplying one or more
+image name regular expressions:
+
+.. code-block:: console
+
+   kayobe overcloud container image build --push ironic- nova-api
+
+Deploy latest CentOS Stream 8 images
+------------------------------------
+
+Make sure you deploy the latest CentOS Stream 8 containers prior to
+this migration:
+
+.. code-block:: console
+
+   kayobe overcloud service deploy
+
+Controllers
+===========
+
+Migrate controllers *one by one*, ideally migrating the host with the Virtual
+IP (VIP) last.
+
+Potential issues
+----------------
+
+-  MariaDB had serious issues one time during testing, after the
+   first controller was migrated. The solution in that instance was to
+   restart the container on the two original CS8 hosts. The behaviour
+   has not been observed again when running
+   ``kayobe overcloud database recover`` between migrations. It can't be
+   said for sure whether this is a genuine solution or the bug just
+   hasn’t occurred these times during testing.
+-  Issues have been seen when attempting to backup the MariaDB database,
+   ``mariabackup`` was segfaulting. This was avoided by reverting to an old
+   MariaDB container image by adding the following in
+   ``etc/kayobe/kolla/globals.yml``:
+
+   .. code-block:: yaml
+
+      mariabackup_image_full: "{{ docker_registry }}/stackhpc/rocky-source-mariadb-server:yoga-20230310T170929"
+- When using Octavia load balancers with the OVN provider and OVN DVR FIP is
+  enabled, restarting Neutron causes load balancers with floating IPs to stop
```

Comment:
This also affects LBs with the Amphora provider (the bug title is outdated)

Reply:
done